lib/client.js

```diff
@@ -913,28 +918,24 @@ Client.prototype._setHostListeners = function () {
 };
 
 Client.prototype._warmup = function (callback) {
-  var loadBalancingPolicy = this.options.policies.loadBalancing;
   var self = this;
-  loadBalancingPolicy.newQueryPlan(this.keyspace, utils.emptyObject, function (err, iterator) {
-    if (err) return callback(err);
-    var hosts = utils.iteratorToArray(iterator);
-    utils.timesLimit(hosts.length, 32, function warmupEachCallback(i, next) {
-      var h = hosts[i];
-      var distance = h.getDistance(loadBalancingPolicy);
-      if (distance !== types.distance.local) {
-        //do not warmup pool for remote or ignored hosts
-        return next();
+  var hosts = this.hosts.values();
+  utils.timesLimit(hosts.length, 32, function warmupEachCallback(i, next) {
```

Comment:
Good question, I arbitrarily used `32` when implementing NODEJS-138 to limit parallelism, but any low number would do.
I'll set it in a constant to avoid having to think about `32` and improve readability.